Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -332,7 +338,7 @@ class SignalAttachment: NSObject {
         for utiType in pasteboardUTISet {
             if UTTypeConformsTo(utiType as CFString, kUTTypeText) {
                 hasTextUTIType = true
-            } else {
+            } else if mediaUTISet.contains(utiType) {
                 hasNonTextUTIType = true
```

Comment:
take it or leave it: would it be simpler to just short circuit and return false here, getting rid of the `hasNonTextUTIType` var?